src/components/TriggerEditor/SimpleTriggerForm.tsx

```diff
@@ -97,6 +97,9 @@ const SimpleTriggerForm = ({
   simpleTriggerInterface,
   validationErrors = {},
 }: SimpleTriggerFormProps): React.ReactElement => {
+  const [endpoints, setEndpoints] = useState<string[]>([]);
+  const [filteredEndpoints, setFilteredEndpoints] = useState<string[]>([]);
+  const [inputValue, setInputValue] = useState<string>('/*');
```

Comment:
no need to save all of this. we can remove it and rely on `simpleTriggerInterface`

```typescript
  const endpointList =
      simpleTriggerInterface?.mappings.map((mapping: AstarteMapping) => mapping.endpoint) || [];
```